ccd-definition/CaseEventToComplexTypes/draftSDO-allParties.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AllPartiesDirections",
+    "CaseEventID": "draftSDO",
+    "CaseFieldID": "allParties",
+    "ListElementCode": "readOnly",
+    "EventElementLabel": " ",
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "READONLY"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AllPartiesDirections",
+    "CaseEventID": "draftSDO",
+    "CaseFieldID": "allParties",
+    "ListElementCode": "directionRemovable",
+    "EventElementLabel": " ",
+    "FieldDisplayOrder": 2,
+    "DisplayContext": "READONLY"
+  },
```

Comment:
Out of interest - are those required to be here? The one which we don't display anyway. Are they just required to satisfy the field show rule?

Reply:
yep, if not defined here the importer throws an error due to not knowing the field

Reply:
it's a shame the amount of duplication that is required for CaseEventToComplexTypes :(

Reply:
Sad days 😭